controls/roles/validator-import-api/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+---
+- name: Waiting for the services to start properly
+  pause:
+    minutes: 5
```

Comment:
This should not be in here. If we need a wait after the `prepare.yaml` then the `pause` should also be in `prepare.yaml`.

Reply:
This task (Pause) is removed from "main.yml" and added into corresponding "prepare.yml"